diesel/src/macros/mod.rs

```diff
@@ -989,6 +989,17 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// assert_eq!(implicit_on_clause_sql, explicit_on_clause_sql);
 /// # }
+///
+/// In the example above, in line `joinable!(posts -> users (user_id));` specifies the relation of
+/// the tables and the ON clause in the following way:
+///
+/// `table_with_foreighn_key -> table_that_foreign_key_points_to (foreign_key)`
+/// and the Sql generated is:
```

Comment:
SQL should be capitalized.

Reply:
Do you think it'd make sense to show how this would be accomplished without `joinable!` instead of demonstrating in terms of generated SQL? e.g. mentioning that `users.inner_join(posts)` becomes equivalent to `users.inner_join(posts.on(users::id.eq(posts::user_id)))`?

Reply:
I think the SQL terms would help people map diesel syntax to something they already know, where the other way would already require a familiarity with diesel. Though both could probably be mentioned.